org.eclipse.gef.examples.shapes/src/org/eclipse/gef/examples/shapes/ShapesEditor.java

```diff
@@ -254,8 +254,9 @@ public void execute(final IProgressMonitor monitor) {
 
```

Comment:
Sadly, GitHub doesn't let me add comments to unmodified code... But in line 166, SimpleFactory is still a raw type. Given that we work on objects there, I'd suggest something like:
```
return new SimpleFactory<>((Class<?>) template);
```

Reply:
I was not sure if Ii would do it. So thx for the feedback fixed it.